src/Diamante/DistributionBundle/Command/InstallCommand.php

```diff
@@ -144,7 +143,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $output->writeln('<info>Installing DiamanteDesk.</info>');
 
-        $this->checkRequirementsStep($output);
+        $this->commandExecutor->runCommand(
```

Comment:
Should be run with -vv
vendor/oro/platform/src/Oro/Bundle/InstallerBundle/Command/InstallCommand.php:256